build_projects/dotnet-cli-build/project.json

```diff
@@ -13,15 +13,27 @@
     "System.Xml.XmlSerializer": "4.0.11-rc2-24027",
     "WindowsAzure.Storage": "6.2.2-preview",
 
-    "Microsoft.DotNet.Cli.Build.Framework": {"target":"project"},
+    "Microsoft.DotNet.Cli.Build.Framework": "1.0.0-*",
     "shared-build-targets-utils": {"target": "project"}
   },
   "frameworks": {
     "netcoreapp1.0": {
       "imports": [
         "dnxcore50",
-        "portable-net45+win8"
+        "portable-net45+win8",
+        "netstandard1.3"
```

Comment:
I don't think this is necessary.  netcoreapp references netstandard1.3 automatically.